nvflare/utils/cli_utils.py

```diff
@@ -196,6 +197,14 @@ def check_job_templates_dir(job_temp_dir: str):
     if job_templates_dir:
         check_job_templates_dir(job_templates_dir)
 
+    if not job_templates_dir:
+        # get default template from nvflare wheel if package installed
+        job_module, import_job_ok = optional_import("nvflare.tool", name="job")
```

Comment:
A bit confused of this line, when will "import_job_ok" be False?
I thought the nvflare.tool.job is always included in our wheel?